YouVsVirus/Assets/Scripts/levelcollectmasks/ButtonHandleLevelMasks.cs

```diff
@@ -21,17 +21,4 @@ public void Continue()
         // Continue to the disco level
         SceneManager.LoadScene("StartScreenLeveldisco_1");
     }
-
-    /// <summary>
-    /// The CanvaslAllCollectedGetHome canvas shows up when the
-    /// player has all masks and gets to the hospital.
-    /// The game is paused.
-    /// This function is called upon pressing space.
-    /// It will  deactivate the canvas and unpause the game.
-    /// </summary>
-    public void UnshowCanvasAllCollectedGetHomeAndContinue()
-    {
-        CanvaslAllCollectedGetHome.SetActive(false);
-        PauseGame.Unpause();
-    }
```

Comment:
Where is this now?

Reply:
I think it is now done in SpaceToContinue.cs